packages/one-app-bundler/utils/getConfigOptions.js

```diff
@@ -11,12 +11,11 @@
  * or implied. See the License for the specific language governing permissions and limitations
  * under the License.
  */
+import { readPackageUpSync } from 'read-pkg-up';
```

Comment:
could be good opportunity to update to r`ead-package-up`

Reply:
is one inherently better than the other?